### ISSUE TITLE
Error if nesting under horizontal-stack

### ISSUE THREAD

**Issue by cpiekacz** [kalkih/mini-graph-card#11]
When I nest `type: custom:mini-graph-card` under the `type: horizontal-stack` I get an error:

> Cannot read property 'states' of undefined

Is it a problem with this component or HA 0.81 version?

**Comment by kalkih** [comment]
Yes, this is an issue introduced with HA 0.81.0 home-assistant/home-assistant-polymer#1904.
Try updating to 0.81.1, worked for me at least, or wait for HA 0.81.2.

**Comment by kalkih** [comment]
@cpiekacz Did you get it working?

**Comment by jonare77** [comment]
Im getting this when stacking the cards horizontal on 0.81.6

![stack error](https://user-images.githubusercontent.com/8417277/48261497-99305200-e41f-11e8-8bf9-d4a08e4c07cc.JPG)

**Comment by kalkih** [comment]
That is funny looking. 😆
I will fix this, however I recommend using something like the glance card instead of this card when stacking that many entities.

**Comment by jonare77** [comment]
Ok, will do, Im a beginner so just trying out.

Mvh
Jon Are Endrerud
Drift, IT Enheten
Ringerike Kommune
Mob: 97531690



On Fri, Nov 9, 2018 at 3:57 PM +0100, "Karl Kihlström" <notifications@github.com<mailto:notifications@github.com>> wrote:


That is funny looking. 😆
I will fix this, however I recommend using something like the glance card instead of this card when stacking that many entities.
